pom.xml

```diff
@@ -303,6 +306,12 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.oracle.ojdbc</groupId>
+                <artifactId>ojdbc8</artifactId>
+                <version>${dep.oracle.version}</version>
```

Comment:
in the dependency management we add only libs that are shared across modules
for single-use deps, we add them directly in the module/pom.xml

similarily, you don't need `dep.` property, if it's just used once

Reply:
cc @ebyhr (i realize it conflicts with yours https://github.com/prestosql/presto/pull/1959#discussion_r343469061)

Reply:
Sorry, my comment was wrong. I referenced mysql and postgresql, but I forgot the fact it is used from other modules.

Reply:
done